app/src/main/res/values/strings.xml

```diff
@@ -2,4 +2,6 @@
     <string name="app_name">EstacionAR</string>
     <string name="main_activity_text_view_title">EstacionAR</string>
     <string name="main_activity_button_set_parking">Establecer estacionamientos</string>
+    <string name="main_activity_set_parking_toast_1">Se establecieron</string>
+    <string name="main_activity_set_parking_toast_2">estacionamientos.</string>
```

Comment:
should use a silngle string with parameters

Reply:
Is not a good name set_parking_ toast_1 and set_parking_ toast_1. Because if you are going to have more toast you must not put set_parking_ toast_3, set_parking_ toast_4, set_parking_ toast_5.....

Reply:
should use a silngle string with parameters